packages/react-tinacms-github/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "react-tinacms-github",
+  "version": "0.0.1-canary3",
+  "description": "",
+  "main": "dist/index.js",
+  "scripts": {
+    "watch": "tsdx watch",
+    "build": "tsdx build",
+    "dev": "tsdx build",
+    "test": "tsdx test --env=jsdom --passWithNoTests",
+    "lint": "tsdx lint"
+  },
+  "author": "",
+  "license": "Apache-2.0",
+  "peerDependencies": {
+    "react": ">=16.8",
+    "@tinacms/styles": "^0.4.0",
+    "@tinacms/fields": "^0.9.1",
+    "styled-components": ">=4.1",
+    "tinacms": ">=0.13",
+    "js-cookie": "^2.2.1",
+    "final-form-submit-listener": "^1.0.0",
+    "@tinacms/icons": ">=0.7.0",
+    "next-tinacms-markdown": ">=0.2.9",
+    "btoa": ">=1.2.0",
```

Comment:
I think a few of these could be `dependencies` instead of `peerDependencies`:

* `js-cookie`
* `final-form-submit-listener`
* `btoa` 

Maybe even `next-tinacms-markdown`